api/proto/teleport/legacy/types/types.proto

```diff
@@ -6074,6 +6074,28 @@ message PluginOAuth2AuthorizationCodeCredentials {
 // PluginStatus is the user-facing status for the plugin instance.
 message PluginStatusV1 {
   PluginStatusCode code = 1;
+  // error_message is a human-readable error message that can be displayed to the user.
+  string error_message = 2;
+  // last_sync_time is the last time the plugin was run.
+  google.protobuf.Timestamp last_sync_time = 4 [
```

Comment:
is field number 3 missing intentionally?

Reply:
Do we want to have "last successful sync time" + "last attempted sync (possibly failed) time"?